src/emulator/storage/files.ts

```diff
@@ -539,9 +527,13 @@ export class StorageLayer {
     const metadataDirPath = path.join(storageExportPath, "metadata");
     await fse.ensureDir(metadataDirPath);
 
-    for await (const [p, file] of this._files.entries()) {
-      const metadataExportPath = path.join(metadataDirPath, encodeURIComponent(p)) + ".json";
-
+    for await (const [, file] of this._files.entries()) {
```

Comment:
do we want to move the copy on L524 to this loop as well? Seems like we can avoid copying untracked files if we do

Reply:
Yeah I think that makes a lot of sense willdo